sdc/tests/test_sdc_numpy.py

```diff
@@ -157,20 +172,57 @@ def sdc_impl(a):
             with self.subTest(data=case):
                 np.testing.assert_array_equal(sdc_func(a), ref_impl(a))
 
-    def test_sum(self):
+    def test_copy(self):
         def ref_impl(a):
-            return np.sum(a)
+            return np.copy(a)
 
         def sdc_impl(a):
-            return numpy_like.sum(a)
+            return numpy_like.copy(a)
 
         sdc_func = self.jit(sdc_impl)
 
-        cases = [[5, 2, 0, 333, -4], [3.3, 5.4, np.nan, 7.9, np.nan]]
+        cases = [[5, 2, 0, 333, -4], [3.3, 5.4, np.nan, 7.9, np.nan], [True, False, True], ['a', 'vv', 'o12oo']]
         for case in cases:
             a = np.array(case)
             with self.subTest(data=case):
                 np.testing.assert_array_equal(sdc_func(a), ref_impl(a))
 
+    def test_copy_int(self):
+        def ref_impl():
+            a = np.array([5, 2, 0, 333, -4])
+            return np.copy(a)
+
+        def sdc_impl():
+            a = np.array([5, 2, 0, 333, -4])
+            return numpy_like.copy(a)
+
+        sdc_func = self.jit(sdc_impl)
+        np.testing.assert_array_equal(sdc_func(), ref_impl())
+
+    def test_copy_bool(self):
+        def ref_impl():
+            a = np.array([True, False, True])
+            return np.copy(a)
+
+        def sdc_impl():
+            a = np.array([True, False, True])
+            return numpy_like.copy(a)
+
+        sdc_func = self.jit(sdc_impl)
+        np.testing.assert_array_equal(sdc_func(), ref_impl())
+
+    @unittest.skip('Numba have not string array')
```

Comment:
```suggestion
    @unittest.skip('Numba doesn't have string array')
```

Reply:
Why we have this tests?

Reply:
This is so that we add functionality if it becomes possible to create a string array in the jit region